toto/models/matchrule.py

```diff
@@ -0,0 +1,102 @@
+"""
+<Module Name>
+  toto/models/validators.py
+
+<Author>
+  Santiago Torres-Arias <santiago@nyu.edu>
+
+<Started>
+  Nov 18, 2016
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  This module provides functions to ensure the syntax of the matchrules is
+  correct.
+
+"""
+from toto.ssl_commons.exceptions import FormatError
+import toto.ssl_crypto.formats
+
+def _validate_match_rule(keywords):
+  """ private helper to verify the syntax of the MATCH matchrule """
+  MATERIAL_OR_PRODUCT = {'PRODUCT', 'MATERIAL'}
```

Comment:
Is there a reason why you initialize this as a `set`? If so, I'd prefer to not use the literal set notation. I think it's easily confused with a dictionary.